pyccl/core.py

```diff
@@ -1023,3 +1023,32 @@ def _set_linear_power_from_arrays(self, a_array=None, k_array=None,
             self.a_array = a_array
             self.k_array = k_array
             self.pk_array = pk_array
+
+
+class CosmologyPlanck18LCDM(Cosmology):
+    """A cosmology roughly corresponding to the best-fit parameters from
+    the 2018 Planck release (arXiv:1807.06209).
+
+    Args:
+        **kwargs (dict): a dictionary of parameters passed as arguments
+            to the `Cosmology` constructor. It should not contain any of
+            the LambdaCDM parameters (`"Omega_c"`, `"Omega_b"`, `"n_s"`,
+            `"sigma8"`, `"A_s"`, `"h"`), since these are fixed.
+    """
+    def __init__(self, **kwargs):
+        h = 0.6732
+        ob = 0.022383
+        oc = 0.12011
+        ns = 0.96605
+        s8 = 0.8120
+        p = {'h': h,
```

Comment:
These are the values @erminiacalabrese should check

Reply:
these are correct if you want the official Planck 2018 best-fit parameters from the nominal combo

Reply:
Awesome, thanks!

Reply:
Planck also assumes a non-zero neutrino mass in their definition of the "base LCDM" model.  It is fixed to M_nu = 0.06 eV.  There is a file distributed with CLASS that contains all the default assumptions:
https://github.com/lesgourg/class_public/blob/master/base_2018_plikHM_TTTEEE_lowl_lowE_lensing.ini
(although the way this is implemented yields N_eff = 3.044 rather than 3.046)

Reply:
Ah good point @jcolinhill 
So, not sure what to do here. On the one hand, we could ignore neutrinos (and maybe remove the `Planck18` from the class name to avoid confusion), since the main point of this class is just to provide a quick way to get a LCDM-like cosmology.
On the other, we can put in neutrinos (this may require more work, to make sure N_eff is right and all that), and that way we get Planck18 proper. However, certain things that we don't support for cosmologies with neutrinos won't work, which will be confusing.
I lean towards the first option. Thoughts @slosar ?

Reply:
I think Planck should have mnu = 0.06 and N_eff  3.046, because 0.002 REALLY does not matter. And for things that are bad we issue warnings anyway.
I also think we should have a VanillaCosmology, which is just h=0.68, Om=0.3, mnu=0 cosmology. Here everything is rounded to make it clear that it is just an approximate quick cosmology and mnu=0 will make most of boltzmann calculations faster.

Reply:
OK, I think we should only have one of those. I'm happy to go for "CosmologyVanilla".

Reply:
(because of the title of this PR)